docs/commands/add.md

```diff
@@ -5,40 +5,64 @@ title: modular add
 
 # `modular add <packageName>`
 
-Adds a new package by creating a new workspace at `packages/<packageName>`,
-omitting the scope if the package is
+Adds a new package by creating a new package at the workspace located at
+`packages/<packageName>`, omitting the scope if the package is
 [scoped](https://docs.npmjs.com/cli/v8/using-npm/scope). If `--path <somePath>`
-is specified, create the workspace at `<somePath>/<packageName>`.
+is specified, the command creates the workspace at `<somePath>/<packageName>`.
 
 (i.e. `modular add my-app` would create a package in `packages/my-app`,
 `modular add @scoped/my-scoped-app` would create a package in
 `packages/my-scoped-app` and `modular add lib-a --path libs` would create a
 package in `libs/lib-a`)
 
-Packages can currently be one of the following types:
+The `modular add` command prompts the user to choose the Modular `type` of the
+package it's about to create. The next section briefly describes the various
+types that can be created by the `modular add` command. For an in-depth
+discussion of the available package types and their characteristics, please see
+[this page](../concepts/package-types.md).
 
-- A standalone `app`. This corresponds to a static Single Page Application (SPA)
-  project in a workspace. Inside this workspace, you can import packages from
-  other workspaces freely, and features like jsx and typechecking work out of
-  the box.
+### Standalone (bundled) package types
```

Comment:
The docs written here covering package types should be moved into the `docs/package-types/index.md` file as I have proposed in another comment. The `add.md` page should link to this, but I think the concepts will be clearer if we separate out the commands from the types rather than mixing them rather thoroughly as we have done.

Reply:
Types are here to provide quick info to developers; new users will run `modular add` for the first time and, when prompted for package type, will go to the "add" command documentation since that's what they run. I think it makes sense to provide a short summary of each type here. See it as a redundancy feature to quickly give the information needed.

I added links to each of the type pages in the type description; it should be easy for users to follow the link if they want more info.